### DISCUSSION TITLE
How to post tweets to a different account, not developer's?

### DISCUSSION THREAD

**Discussion by GuanacoDevs** [tweepy/tweepy#1273] [Questions] [answered]
Currently I'm using `tweepy` to `update_status` of my developer's account, it works great.
Now I'm about to deploy a different app and I want to do the same, how can I use `tweepy` to post to that account?, is it possible?

UPDATE: I found a way to do it with `twython`, however I'd like to stick with `tweepy`, I'll close the issue but I'd really appreciate if someone could point me in the right documentation to do it with **`tweepy`**

Best Regards

**Comment by Harmon758** [comment, marked as answer]
You need to authenticate as the other user.
See https://tweepy.readthedocs.io/en/latest/auth_tutorial.html, https://developer.twitter.com/en/docs/basics/authentication/overview/oauth, and https://developer.twitter.com/en/docs/basics/authentication/overview/3-legged-oauth (or https://developer.twitter.com/en/docs/basics/authentication/overview/pin-based-oauth).
